sycl/test/plugins/sycl-ls-gpu-opencl.cpp

```diff
@@ -3,8 +3,8 @@
 // RUN: env SYCL_BE=PI_OPENCL sycl-ls --verbose >%t.opencl.out
 // RUN: FileCheck %s --check-prefixes=CHECK-GPU-BUILTIN,CHECK-GPU-CUSTOM --input-file %t.opencl.out
 
-// CHECK-GPU-BUILTIN: gpu_selector(){{.*}}GPU : OpenCL
-// CHECK-GPU-CUSTOM: custom_selector(gpu){{.*}}GPU : OpenCL
+// CHECK-GPU-BUILTIN: gpu_selector(){{.*}}GPU : 2.1
```

Comment:
we need to check format instead of exact values. e.g. {{[0-9].[0-9]}}
Also we need to be sure that it works for CUDA RT and other OpenCL/L0 RTs as well as HOST device. May be additional test can be aded/updated.